Worker.Extension.Sql/src/SqlInputAttribute.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Azure.Functions.Worker.Extensions.Abstractions;
+
+namespace Microsoft.Azure.Functions.Worker.Extension.Sql
+{
+    public sealed class SqlInputAttribute : InputBindingAttribute
```

Comment:
So why does the out of proc have separate attributes for input/output but in-proc just has a single one that's shared? @fabiocav is there a reason you'd know of why it seems to be mostly done like this across all the extensions?